Adding campaign republish behavior docs (#561)
mautic/user-documentation: docs/campaigns/creating_campaigns.rst

-  **Category** - Choose a Category to assign your Campaign to. Categories help you organize your Campaigns. To learn more about creating and managing Categories, see :doc:`/categories/categories-overview`.
-  **Allow Contacts to restart the Campaign** - Click the toggle switch to allow Contacts to restart the Campaign if you're building a Campaign for a recurring message - for example birthdays, subscriptions - or transactional operations - for example activity notifications, updating data. Enabling this option allows Contacts to go through the same Campaign multiple times.

> -  **Campaign Reactivation Behaviour** - Configure how scheduled events with relative delays should behave when the Campaign is reactivated after being deactivated. You can override the global default setting for this specific Campaign. See :ref:`Campaign Reactivation Behaviour<campaign reactivate behavior>` for more information about the available options.

Reviewer: Would the meaning of explanation changed if we use active voice as below?

```suggestion
-  **Campaign Reactivation Behaviour** - Configure how scheduled events with relative delays should behave when you reactivate a deactivated Campaign. You can override the global default setting for this specific Campaign. See :ref:`Campaign Reactivation Behaviour<campaign reactivate behavior>` for more information about the available options.
```
